lib/experimental/Information/Avatars/BaseAvatar/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  color as AvatarColors,
+  Avatar as AvatarComponent,
+  AvatarFallback,
+  AvatarImage,
+} from "@/ui/avatar"
+import { ComponentProps, forwardRef } from "react"
+
+type ShadAvatarProps = ComponentProps<typeof AvatarComponent>
+
+type Props = {
+  type: ShadAvatarProps["type"]
+  name: string | string[]
+  src?: string
+  size?: ShadAvatarProps["size"]
+  color?: ShadAvatarProps["color"] | "random"
+}
+
+function getInitials(
+  name: string | string[],
+  size?: ShadAvatarProps["size"]
+): string {
+  const nameArray = Array.isArray(name) ? name : [name]
+  const hasSmallSize = size === "xsmall" || size === "small"
```

Comment:
I propose to reduce some redundancy in the name, because _small_ is always about size.

```suggestion
  const isSmall = size === "xsmall" || size === "small"
```